docs/conf.py

```diff
@@ -131,10 +131,6 @@
 maximum_signature_line_length = 90
 sphinxemoji_style = "twemoji"
 
-suppress_warnings = [
-    "autosummary.import_cycle",
-]
```

Comment:
The `autosummary.import_cycle` warnings no longer appeared, and the new warnings from this PR actually did refer to a real problem with indentation in docstrings created by the decorator for `to_hz` that we didn't get warnings about before.